rootbeerlib/src/main/java/com/scottyab/rootbeer/Const.java

```diff
@@ -77,6 +80,34 @@ private Const() throws InstantiationException {
             //"/dev"
     };
 
+    /**
+     * Get a list of paths to check for binaries
+     *
+     * @return List of paths to check, using a combination of a static list and those paths
+     * listed in the PATH environment variable.
+     */
+    static String[] getPaths(){
+        ArrayList<String> paths = new ArrayList<>(Arrays.asList(suPaths));
+
+        String sysPaths = System.getenv("PATH");
```

Comment:
Simple as that? didn't know there's access to this from app context

Reply:
yup, pretty handy :)